networking/tests/common.rs

```diff
@@ -0,0 +1,188 @@
+use std::slice::Chunks;
```

Comment:
@akoptelov 
every source file should start with:
```
// Copyright (c) SimpleStaking and Tezedge Contributors
// SPDX-License-Identifier: MIT
```
please check all new added files here also